docs/puzzle/wahgyu_stakes.html

```diff
@@ -317,39 +319,10 @@ <h1>
 }
 
 </style>
-<script type="text/javascript">
-$("#move_form").on("submit", function(event){
-  event.preventDefault();
-  $.ajax({
-    url: $(this).attr("action"),
-    type: $(this).attr("method"),
-    data : $(this).serialize(),
-    success : function(json) {
-      $("#move_response").removeClass()
-      $("#move_response").empty()
-
-      if (json.success) {
-        $("#move_response").append(`<div>${json.response[0]}</div>`)
-        $("#move_response").append(`<div>${json.response[1]}`)
-        if (json.response[2].length > 0) {
-          $("#move_response").append(`<div class="monster_cries">${json.response[2]}</div></div>`)
-        }
-        if (json.response[3].length > 0) {
-          $("#move_response").append(`<div><img src="../static/puzzle_resources/wahgyu_stakes/hp.1134c984e497.gif" class="rem2"></img>${json.response[3]}</div>`)
-        }
-      } else {
-        $("#move_response").append(`<div class="text-danger">${json.response}</div>`)
-      }
-    },
-    error : function(xhr, errmsg, err) {
-      console.log(xhr.status + ": " + xhr.responseText);
-      $("#move_response").removeClass()
-      $("#move_response").addClass("text-danger")
-      $("#move_response").text("Error. Please email contact@huntinality.com if this issue persists.")
-    }
-  });
-});
+<script id="check_moves" type="text/javascript">
+  window.onload = move_checker_fn()
```

Comment:
Is this refactor just for better organization? Or did we have to move stuff out of this file for other reasons?

Reply:
The old code relied on there being a server to do the answer checking. I moved it out of this file, so that the moves wouldn't be indexed by search engine.

Reply:
sgtm 👍